modules/nextflow/src/main/groovy/nextflow/cli/CmdClone.groovy

```diff
@@ -15,6 +15,9 @@
  */
 
 package nextflow.cli
+
+import static nextflow.scm.AssetManager.revisionDelim
```

Comment:
By convention constanta are upper case and separated by `_`

Reply:
thanks, fixed